app/assets/javascripts/reactivation_warning.js

```diff
@@ -0,0 +1,9 @@
+$(document).on('turbolinks:load', function(){
+  $('input#passenger_active').change(function(){
+      if($(this).is(':checked')){
+        alert("You are about to temporarily reactivate this passenger. \
+              \n Please note that, unless an updated doctor's note is \
+              \n obtained, this passenger will deactivate again at 4:30am.");
```

Comment:
I don't want this reactivation warning unless I didn't update the expiration date. Also, instead of "doctor's note" why don't we reference the expiration date instead? An updated doctor's note could mean anything.